### DISCUSSION TITLE
Help with installation

### DISCUSSION THREAD

**Discussion by KristofSterkendries** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#135] [Q&A]
Anyone here present can help me with the installation? Whenever I get the github link, it stays stuck at 6%. (75/1204 objects).

**Comment by JafarAkhondali** [comment]
That's weird, try other methods such as download as Zip or ssh cloning.

**Reply by KristofSterkendries** [reply]
Gives me this error : 

git clone https://github.com/JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module
Cloning into 'acer-predator-turbo-and-rgb-keyboard-linux-module'...
remote: Enumerating objects: 1204, done.
remote: Counting objects: 100% (178/178), done.
remote: Compressing objects: 100% (71/71), done.
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
error: 7497 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

**Reply by KristofSterkendries** [reply]
Ok @JafarAkhondali  - I figured it out. 
As I am completely new mind just writing me the script to put in to keep my keyboard lit up in the red color as bright as possible at all times, please? :-)

A dononation will follow if it works. :package:

**Reply by JafarAkhondali** [reply]
Just try other methods for cloning. Otherwise you can check this thread: https://stackoverflow.com/questions/38618885/error-rpc-failed-curl-transfer-closed-with-outstanding-read-data-remaining

**Reply by KristofSterkendries** [reply]
@JafarAkhondali - I figured out how to do it. :) Just trying to get the keyboard to stay red at all times with full brightness now

**Reply by JafarAkhondali** [reply]
```

#Install as service 
cd "acer-predator-turbo-and-rgb-keyboard-linux-module"
chmod +x ./*.sh
sudo ./install_service.sh

# Make all zones red
python3 facer_rgb.py -m 0 -z 1 -cR 255 -cB 0 -cG 0
python3 facer_rgb.py -m 0 -z 2 -cR 255 -cB 0 -cG 0
python3 facer_rgb.py -m 0 -z 3 -cR 255 -cB 0 -cG 0
python3 facer_rgb.py -m 0 -z 4 -cR 255 -cB 0 -cG 0

```

**Reply by maxie7** [reply]
I've added this  script as a task to 'sudo' crontab

`sudo crontab -l`

You have to have this line in it:

_@reboot sudo /home/YOURUSERNAME/src/ascripts/./startup_script.sh_

Script:

```
#!/bin/zsh

# Navigate to the directory
cd /home/YOURUSERNAME/src/ascripts/acer-predator-turbo-and-rgb-keyboard-linux-module

# Run the installation script with sudo
./install.sh

# Run the RGB script
./facer_rgb.py -m 0 -z 1 -s 0 -b 30 -cR 0 -cB 0 -cG 255
./facer_rgb.py -m 0 -z 2 -s 0 -b 30 -cR 0 -cB 0 -cG 255
./facer_rgb.py -m 0 -z 3 -s 0 -b 30 -cR 0 -cB 0 -cG 255
./facer_rgb.py -m 0 -z 4 -s 0 -b 30 -cR 250 -cB 0 -cG 0
```

P.S. > My laptop is `Predator PHN16-71`
